src/components/LoanCards/LoanUse.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<p class="tw-line-clamp-4">
+		{{ loanUse }}
+	</p>
+</template>
+
+<script>
+import numeral from 'numeral';
+
+export default {
+	name: 'LoanUse',
+	props: {
+		anonymizationLevel: {
+			type: String,
+			default: 'none',
+		},
+		use: {
+			type: String,
+			default: '',
+		},
+		loanAmount: {
+			type: String,
+			default: '',
+		},
+		status: {
+			type: String,
+			default: '',
+		},
+		borrowerCount: {
+			type: Number,
+			default: 1,
+		},
+		name: {
+			type: String,
+			default: '',
+		},
+		loanUseMaxLength: {
+			type: Number,
+			default: 100,
+		},
+	},
+	computed: {
+		helpLanguage() {
+			if (this.status === 'fundraising' || this.status === 'inactive' || this.status === 'reviewed') {
```

Comment:
Could this be a switch?

Reply:
Oh no, is a or condition...

Reply:
It could theoretically be a switch, but when there's only two cases, I usually just keep it an if.